packages/expect/src/state.ts

```diff
@@ -33,3 +34,18 @@ export function setState<State extends MatcherState = MatcherState>(
   Object.assign(current, state)
   map.set(expect, current)
 }
+
+export function getCustomEqualityTesters(): Array<Tester> {
+  const { customTesters } = getState((globalThis as any)[GLOBAL_EXPECT])
+  return customTesters || []
+}
+
+export function addCustomEqualityTesters(testers: Array<Tester>): void {
+  if (!Array.isArray(testers)) {
+    throw new TypeError(
+      `expect.customEqualityTesters should receive array type, but got: ${getType(testers)}`,
+    )
+  }
+
+  setState({ customTesters: [...testers] }, (globalThis as any)[GLOBAL_EXPECT])
```

Comment:
This  breaks compatibility with Jest, there it doesn't override `testers`, but pushes a new one

Reply:
Thanks~ Done